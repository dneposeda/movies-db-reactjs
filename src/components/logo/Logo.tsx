import React from "react";

const Logo = () => (
    <div className="logo">
        <a href="/" className="logo__link">
             <span className="logo__text"><span>netflix</span>roulette</span>
        </a>
    </div>
)

export default Logo;
