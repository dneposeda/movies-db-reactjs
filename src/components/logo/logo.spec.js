import React from 'react';
import Logo from './Logo';

describe('Logo', () => {
    it('renders correctly', () => {
        const component = shallow(
            <Logo />
        );
        expect(component).toMatchSnapshot();
    });
});
