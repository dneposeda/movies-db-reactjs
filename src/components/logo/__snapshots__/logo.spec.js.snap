// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<div
  className="logo"
>
  <a
    className="logo__link"
    href="/"
  >
    <span
      className="logo__text"
    >
      <span>
        netflix
      </span>
      roulette
    </span>
  </a>
</div>
`;
